app/controllers/availabilities_controller.rb

```diff
@@ -0,0 +1,68 @@
+class AvailabilitiesController < ApplicationController
+	def index
+		@availabilities = Availability.where(presenter: current_user.presenter).order('availabilities.start_time ASC')
```

Comment:
It'd be great if you could abstract this query into the `Availability model` to clean up the controller